React Native SDK implementation in Segment-RudderStack migration guide (#447)
rudderlabs/rudderstack-docs: user-guides/migration-guides/rudderstack-migration-guide.mdx

You can use the rest of your code without any changes as the RudderStack API is fully compatible with Segment.
</div>
</li>
</ul>
    </TabPanel>
    <TabPanel>
      <ul>

>       <li>Uninstall the Segment React Native SDK by running the following command:

Author: ```suggestion
      <li>Install the RudderStack React Native SDK by running the following command:
```

<span>

```bash
// npm
npm install @rudderstack/rudder-sdk-react-native --save
// yarn
yarn add @rudderstack/rudder-sdk-react-native
// navigate to iOS folder and run pod install
pod install
```
</span>
      </li>
      <li>Import the Rudderstack React Native SDK as shown below:
<span>

```typescript
import rudderClient from "@rudderstack/rudder-sdk-react-native"
```
</span>
	</li>
	<li>Change the SDK initialization as shown in the following snippet. Replace <code class="inline-code">WRITE_KEY</code> and <code class="inline-code">DATA_PLANE_URL</code> with your source write key and data plane URL obtained above.
<span>

```typescript

> await rudderClient.setup(WRITE_KEY, {

Reviewer: ```suggestion
await rudderClient.setup(<WRITE_KEY>, {
```

>   dataPlaneUrl: DATA_PLANE_URL,

Reviewer: ```suggestion
  dataPlaneUrl: <DATA_PLANE_URL>,
```
